Packs/Cryptosim/Integrations/Cryptosim/Cryptosim.py

```diff
@@ -0,0 +1,226 @@
+from datetime import datetime, timedelta
+from CommonServerPython import *
+from CommonServerUserPython import *
+import traceback
+import json
+import base64
+import requests
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+
+class Client(BaseClient):
+
+    def correlation_alerts(self):
+        args = demisto.args()
+        endTime = datetime.now()
+        startTime = endTime - timedelta(minutes=demisto.params().get('incidentFetchInterval', 360))
```

Comment:
```suggestion
    def correlation_alerts(self, args, end_time=None, start_time=None):
        end_time = datetime.now() if not end_time else end_time
        start_time = end_time - timedelta(minutes=demisto.params().get('incidentFetchInterval', 360)) if not start_time else start_time
```

Reply:
We want that when you are fetching, you will pass the `start_time` by the last incident that you got the last fetch

Reply:
Okey, I changed it with end_time as parameter. And each fetch_incident trigger alerts with end_time. When end_time changed start_time is changed also because it is depending end_time